### ISSUE TITLE
Special characters is not shown in powershell.

### ISSUE THREAD

**Issue by david-minaya** [Jaykul/PowerLine#48]
I install PowerLine like they saying in the README. First install the Pansies module and after install the PowerLine Module and finally import the PowerLine module. But, when I import the PowerLine module I get this.

![powershell](https://user-images.githubusercontent.com/21043752/60769925-90257c80-a0a3-11e9-8167-3a9f4c50606d.png)

I've too try out this in ConEmu

**Comment by bozho** [comment]
You're missing `-PowerLineFont` switch in your `Set-PowerLinePrompt` call

**Comment by Elia0107** [comment]
Hi, when I use the command (for example: Set-PowerLinePrompt -PowerLineFont "Hack" -SetCurrentDirectory -RestoreVirtualTerminal -Newline -Timestamp -Colors "#00DDFF", "#0066FF" ), I get an error like Immpossible find positional parameter with argument "Hack".

**Comment by Jaykul** [comment]
Yeah, `-PowerLineFont` is just a switch which tells PowerLine that you are already using a font with PowerLine characters in it. You don't need to pass any value to it. E.g.:

```posh
Set-PowerLinePrompt -PowerLineFont -SetCurrentDirectory -RestoreVirtualTerminal -Newline -Timestamp -Colors "#00DDFF", "#0066FF"
```

**Comment by Tommasov** [comment]
![Annotation 2020-01-15 184414](https://user-images.githubusercontent.com/9279914/72457533-1b0bd000-37c7-11ea-86cf-723f275fb058.jpg)
I have the same issue, any help?

Thanks
